http-server/src/main/java/org/triplea/db/dao/error/reporting/ErrorReportingDao.java

```diff
@@ -23,4 +26,12 @@
    */
   @SqlUpdate("delete from error_report_history where date_created < :purgeSinceDate")
   void purgeOld(@Bind("purgeSinceDate") Instant purgeSinceDate);
+
+  @SqlQuery(
+      "select created_issue_link"
+          + "  from error_report_history"
+          + "  where report_title = :reportTitle "
+          + "    and game_version = :gameVersion")
```

Comment:
I wonder if a check for equality is the right move here.
If we fix an issue, but people are still using an unpatched version, they shouldn't be able to create an issue IMO.
Thoughts?

Reply:
That'll be a problem if you go back in time. So we can certainly get 'n' issues up until a problem is fixed. That is a risk. If we fail to fix an issue, it's more risky if we suppress it. False-positives are not great,  false-negatives are worse.

Does that make sense?

Reply:
@RoiEXLab how do you mean by an unpatched version? 

Let's say a problem is reported for 'v2', and someone is using a 'v3' and we only have a fix in 'v4', we can get a duplicate report fro both 'v2' and 'v3', is that the scenario you are considering?

IMO it's okay:
- the stack traces could change between v2 and v3 along with line numbers. Having that be different is slightly helpful.
- we still go from 'n' reports being linear to the number of versions instead of number of occurrances.
- we avoid a situation where we think we fixed the problem in 'v4', but perhaps we did not. If we de-dupe on title only, then a closed issue is theoretically definitive even if the problem is re-introduced or never fixed in the first place.

In part we're being clever about detecting when problems are fixed by just de-duping by title and version. If we suppressed all future version reports we'd need to know when a problem is fixed. As-is, a person looks at the bug report, sees it's closed and hopefully we'll comment which version has a fix. 

Going forward pre-release hopefully will get back to being used for preview and testing and not early access like it was. In part the early access was due to map features and because map versioning is deficient when it comes to forward & backward compatibility. Removing the need for early access implies we've some work to improve how maps are handled.

Reply:
>  is that the scenario you are considering

Yes, that's exactly what I meant